### PR TITLE
[Android] Implement custom WebChromeClient in WebViewHandler

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

Implement custom **WebChromeClient** in WebViewHandler.

### PR Checklist ###

- [x] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)

#### Does this PR touch anything that might affect accessibility?
- No

### CODE REVIEW

**Comment by rmarinho** [comment]
@jsuarezruiz  if you check we added Mappers for those properties 

https://github.com/dotnet/maui/blob/main/src/Core/src/Handlers/WebView/WebViewHandler.Android.cs#L54

File: src/Core/src/Handlers/WebView/WebViewHandler.Android.cs [hunk 1]
```diff
@@ -37,10 +50,23 @@ public override void SetVirtualView(IView view)
...
 		{
 			platformView.StopLoading();
+			
+			_webViewClient?.Dispose();
+			_webChromeClient?.Dispose();
 
 			base.DisconnectHandler(platformView);
 		}
 
+		protected virtual WebViewClient GetWebViewClient()
```
Comment (rmarinho) [inline src/Core/src/Handlers/WebView/WebViewHandler.Android.cs:60]: why are we adding this if we have mappers?

Comment (jsuarezruiz) [reply src/Core/src/Handlers/WebView/WebViewHandler.Android.cs:53]: Right, is not necessary. I removed it.

File: src/Core/src/Platform/Android/ActivityResultCallbackRegistry.cs [hunk 2]
```diff
@@ -1,46 +1,49 @@
...
+namespace Microsoft.Maui.Platform
 {
 	public static class ActivityResultCallbackRegistry
 	{
-		static readonly ConcurrentDictionary<int, Action<Result, Intent>> s_activityResultCallbacks =
+		static readonly ConcurrentDictionary<int, Action<Result, Intent>> ActivityResultCallbacks =
 			new ConcurrentDictionary<int, Action<Result, Intent>>();
 
-		static int s_nextActivityResultCallbackKey;
+		static int NextActivityResultCallbackKey;
```
Comment (rmarinho) [inline src/Core/src/Platform/Android/ActivityResultCallbackRegistry.cs:16]: why this change? static fields should be s_ by guidelines

**Review by Redth — APPROVED** [review]
